frontend/components/usage.js

```diff
@@ -0,0 +1,118 @@
+    </p>
+    <table className="centered"><tbody>
+      <tr><td>   Dashes <code>--</code>
+      </td><td>  →
+      </td><td>  <code>-</code> Dash
+      </td></tr>
+      <tr><td>   Underscores <code>__</code>
+      </td><td>  →
+      </td><td>  <code>_</code> Underscore <br />
+      </td></tr>
+      <tr><td>   <code>_</code> or Space <code>&nbsp;</code>
+      </td><td>  →
+      </td><td>  <code>&nbsp;</code> Space
+      </td></tr>
+    </tbody></table>
+
+    <p className="badge-img">
+      <img src={baseUri + "/badge/color-brightgreen-brightgreen.svg"} alt="brightgreen" />
+      <img src={baseUri + "/badge/color-green-green.svg"} alt="green" />
+      <img src={baseUri + "/badge/color-yellowgreen-yellowgreen.svg"} alt="yellowgreen" />
+      <img src={baseUri + "/badge/color-yellow-yellow.svg"} alt="yellow" />
+      <img src={baseUri + "/badge/color-orange-orange.svg"} alt="orange" />
+      <img src={baseUri + "/badge/color-red-red.svg"} alt="red" />
+      <img src={baseUri + "/badge/color-lightgrey-lightgrey.svg"} alt="lightgrey" />
+      <img src={baseUri + "/badge/color-blue-blue.svg"} alt="blue" />
+      <img src={baseUri + "/badge/color-ff69b4-ff69b4.svg"} alt="ff69b4" />
+    </p>
+
+    <h3 id="dynamic-badge">Dynamic</h3>
+
+    <DynamicBadgeMaker baseUri={baseUri} />
+
+    <p>
+      <code>/badge/dynamic/&lt;TYPE&gt;.svg?uri=&lt;URI&gt;&amp;label=&lt;LABEL&gt;&amp;query=&lt;<a href="https://www.npmjs.com/package/jsonpath" target="_BLANK" title="JSONdata syntax">$.DATA.SUBDATA</a>&gt;&amp;colorB=&lt;COLOR&gt;&amp;prefix=&lt;PREFIX&gt;&amp;suffix=&lt;SUFFIX&gt;</code>
+    </p>
+
+    <hr className="spacing" />
+
+    <h2 id="styles">Styles</h2>
+
+    <p>
+      The following styles are available (flat is the default as of Feb 1st 2015):
+    </p>
+    <table className="badge-img"><tbody>
+      <tr>
+        <td><img src={baseUri + "/badge/style-plastic-green.svg?style=plastic"} alt="" /></td>
+        <td><code>https://img.shields.io/badge/style-plastic-green.svg?style=plastic</code></td>
+      </tr>
+      <tr>
+        <td><img src={baseUri + "/badge/style-flat-green.svg?style=flat"} alt="" /></td>
+        <td><code>https://img.shields.io/badge/style-flat-green.svg?style=flat</code></td>
+      </tr>
+      <tr>
+        <td><img src={baseUri + "/badge/style-flat--square-green.svg?style=flat-square"} alt="" /></td>
+        <td><code>https://img.shields.io/badge/style-flat--square-green.svg?style=flat-square</code></td>
+      </tr>
+      <tr>
+        <td><img src={baseUri + "/badge/style-for--the--badge-green.svg?style=for-the-badge"} alt="" /></td>
+        <td><code>https://img.shields.io/badge/style-for--the--badge-green.svg?style=for-the-badge</code></td>
+      </tr>
+      <tr>
+        <td><img src={baseUri + "/badge/style-social-green.svg?style=social"} alt="" /></td>
+        <td><code>https://img.shields.io/badge/style-social-green.svg?style=social</code></td>
```

Comment:
There's a lot of repetition here. Consider iterating over an array, roughly like:
```
const styles = [
  'plastic',
  'flat',
  'flat-square',
  ......
];

<table className="badge-img"><tbody>
{styles.map(style => (
      <tr>
        <td><img src={baseUri + '/badge/style-' + style + '-green.svg?style=' + style} alt="" /></td>
        <td><code>https://img.shields.io/badge/style-{style}-green.svg?style={style}</code></td>
      </tr>
))}
</tbody></table>
```